datahub-web-react/src/app/onboarding/onboarding.tsx

```diff
@@ -0,0 +1,693 @@
+                    DataHub supports OIDC SSO.
+                </p>
+                <p>
+                    Learn more about how to configure SSO{' '}
+                    <a
+                        target="_blank"
+                        rel="noreferrer noopener"
+                        href="https://datahubproject.io/docs/authentication/guides/sso/configure-oidc-react/#configuring-oidc-in-react"
+                    >
+                        {' '}
+                        here.{' '}
+                    </a>
+                </p>
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: USERS_INVITE_LINK_ID,
+        selector: `#${USERS_INVITE_LINK_ID}`,
+        title: 'Sending Invite Links',
+        content: (
+            <Typography.Paragraph>
+                <p>
+                    You can also generate invite links to onboard new <strong>Users</strong> here, with different invite
+                    links for the different <strong>Roles</strong> you want users to assume.
+                </p>
+                <p>
+                    Learn more about configuring invite links{' '}
+                    <a
+                        target="_blank"
+                        rel="noreferrer noopener"
+                        href="https://datahubproject.io/docs/authentication/guides/add-users/#send-prospective-users-an-invite-link"
+                    >
+                        {' '}
+                        here.{' '}
+                    </a>
+                </p>
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: USERS_ASSIGN_ROLE_ID,
+        selector: `#${USERS_ASSIGN_ROLE_ID}`,
+        title: 'Assigning Roles',
+        content: (
+            <Typography.Paragraph>
+                <p>
+                    You can also assign <strong>Roles</strong> to existing <strong>Users</strong> here.
+                </p>
+                <p>
+                    Learn more about <strong>Roles</strong>{' '}
+                    <a
+                        target="_blank"
+                        rel="noreferrer noopener"
+                        href="https://datahubproject.io/docs/authorization/roles"
+                    >
+                        {' '}
+                        here.{' '}
+                    </a>
+                </p>
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: GROUPS_INTRO_ID,
+        title: 'Groups Page',
+        content: (
+            <Typography.Paragraph>
+                <p>
+                    Welcome to the <strong>Groups</strong> page!
+                </p>
+                <p>
+                    <strong>Groups</strong> are collections of users which can be used to assign ownership to assets and
+                    configure authorization.
+                </p>
+                <p>
+                    <strong>Groups</strong> can be created natively via the DataHub UI, or synced from your Identity
```

Comment:
within DataHub. Please remove ALL references to "UI". For most users, the UI ==== DataHub. They don't interact with anything else. I want to kill our public usage of "UI".

Reply:
Thankfully this was the only one!